src/openvic-simulation/pop/Pop.cpp

```diff
@@ -162,6 +172,59 @@ void Pop::update_gamestate(
 //TODO store income
 void Pop::add_rgo_owner_income(const fixed_point_t income) {}
 void Pop::add_rgo_worker_income(const fixed_point_t income) {}
+//adds income to the temporary stores and then to general income store
+void Pop::add_artisanal_income(const fixed_point_t income){
+	artisanal_income += income;
+	this->income += income;
+}
+void Pop::add_factory_worker_income(const fixed_point_t income){
+	this->factory_worker_income += income;
+	this->income += income;
+}
+void Pop::add_factory_owner_income(const fixed_point_t income){
+	this->factory_owner_income += income;
+	this->income += income;
+}
+void Pop::add_unemployment_subsidies_income(const fixed_point_t income){
```

Comment:
I'd drop the _income for unemployment_subsidies, pensions and government_salary.